test/image_test.py

```diff
@@ -1099,20 +1099,20 @@ def get_hash(img: Image) -> str:
     assert get_hash(img) == "385413df75dffe57f41d4c8eef45ce6cec6de54dda348f3b93c3b7d36fdf0973"
 
     img = Image.from_registry("ubuntu:22.04")
-    assert get_hash(img) == "40ff7675fd5ecca1d851887c6a3553008704f901748de1fb3c02035eb9c46f56"
+    assert get_hash(img) == "aa4b17f73658c5ae09ca8dfce9419cd50c6179ecf015208151cc2c7109ed8e40"
```

Comment:
Yeah the latest Image Builder is still in "preview" status. Also this only affects `Image.from_registry` / `Image.from_dockerfile`; we don't use the `python -m pip` spelling in `Image.debian_slime`, I guess because we know there's only going to be one Python installation there.